ledger/src/shred/merkle_tree.rs

```diff
@@ -9,6 +9,7 @@ pub(crate) const SIZE_OF_MERKLE_ROOT: usize = std::mem::size_of::<Hash>();
 const_assert_eq!(SIZE_OF_MERKLE_ROOT, 32);
 const_assert_eq!(SIZE_OF_MERKLE_PROOF_ENTRY, 20);
 pub(crate) const SIZE_OF_MERKLE_PROOF_ENTRY: usize = std::mem::size_of::<MerkleProofEntry>();
+pub(crate) const MAX_MERKLE_PROOF_ENTRIES: u8 = 6;
```

Comment:
Is this for 32:32 or for 64:64 batches we have today?

Reply:
this is for 32:32 batches. I want to keep this value, but I will rename things and add comments to make things clear